SOURCES/WEBAPP/ESP32/aurora/hwconfig.h

```diff
@@ -33,15 +33,16 @@
 #define HAVE_IRRECEIVER (1)
 #define IR_RECEIVER_PIN 5
 
-#define APPLE_REMOTE_UP          0x77E1D047
-#define APPLE_REMOTE_DOWN        0x77E1B047
-#define APPLE_REMOTE_LEFT        0x77E11047
-#define APPLE_REMOTE_RIGHT       0x77E1E047
-#define APPLE_REMOTE_CENTER_DOWN 0x77E1BA47
-#define APPLE_REMOTE_CENTER_UP   0x77E12047
-#define APPLE_REMOTE_MENU        0x77E14047
-#define APPLE_REMOTE_PLAY_DOWN   0x77E17A47
-#define APPLE_REMOTE_PLAY_UP     0x77E12047
+#define APPLE_REMOTE_UP          0x77E1D055
```

Comment:
https://github.com/brackendawson/Appleceiver/blob/357a0a16013d9e35f1119530399aba55b7030bab/Appleceiver.ino#L43-L55 suggests it should be possible to support multiple Apple remotes using correct bit mask.

Reply:
Totally agree. But I would like to open another PR for this.